### PR TITLE
Updated solana commands

### PR DESCRIPTION
**Pull request by chamoisthefirst**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1126,18 +1128,22 @@ client.on("messageCreate", async (message) => {
 
     let numSolana = parseInt(args[0]);
 
+    if(args[0] === "all"){
+      numSolana = Math.floor(storage[message.author.id].money/costOfSolana);
```
Comment (MinecraftXPert) [inline index.js:1129]: Just add an embed here and it should be good

**Review by MinecraftXPert — COMMENTED** [review]
Just add something

File: index.js (hunk 1 above)
Comment (chamoisthefirst) [reply index.js:1129]: aight

**Comment by HotelNimara** [comment]
image


I finished some commands and now I don't know what ideas I have for the future.

Maybe there are other ideas?

**Pull request closed without merging**